docker/nvidia/docker-compose.template.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  dtk_dev:
+    extends:
+      file: ../docker-compose.yml
+      service: dtk_dev
+    image: dtk_dev_cuda
```

Comment:
`../docker-compose.yaml` has `image` = `dalg24/dtk-stack:latest`. Why does this one not follow the convention?